apps/angular/src/core/mappers/anime.mapper.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+
+import { AnimeDto } from '@js-camp/core/dtos/anime';
+import { AnimeMapper as BaseAnimeMapper } from '@js-camp/core/mappers/anime';
+import { Anime } from '@js-camp/core/models/anime';
+
+/** Anime mapper. */
+@Injectable({ providedIn: 'root' })
+export class AnimeMapper {
+	/**
+	 * Mapping from dto to domain model.
+	 * @param anime - The AnimeDto object to be converted.
+	 */
+	public fromDto(anime: AnimeDto): Anime {
+		return BaseAnimeMapper.fromDto(anime);
+	}
+}
```

Comment:
Writing it here, but please apply to all jsdoc in the PR. Please omit `-` after param.
Also, DTO = data transfer object, so writing "AnimeDto object" is a duplication =)

```suggestion
	/**
	 * Mapping from dto to domain model.
	 * @param dto Anime DTO.
	 */
	public fromDto(dto: AnimeDto): Anime {
		return BaseAnimeMapper.fromDto(dto);
	}
}
```

Reply:
Haha thanks brother, not only learn code but also learn English 😆😆